src/services/token.js

```diff
@@ -116,16 +116,23 @@ export class Service {
     const secret = this.options.secret;
     const options = {
       algorithm,
-      expiresIn,
       notBefore,
       audience,
-      issuer,
       jwtid,
-      subject,
       noTimestamp,
       header
     };
 
+    if (!data.iss) {
```

Comment:
They are already getting passed when you login with a token, which is
causing an error.
On Wed, Aug 3, 2016 at 8:37 PM Eric Kryski <notifications@github.com> wrote:

> In src/services/token.js
> <https://github.com/feathersjs/feathers-authentication/pull/255#discussion_r73454675>
> :
>
> >        noTimestamp,
> >        header
> >      };
> >
> > +    if (!data.iss) {
>
> Confused a bit about this one @marshallswain
> <https://github.com/marshallswain>. Maybe I'm missing something. Any
> reason you want to pass those as data?
>
> I was thinking any token options should come from params.jwt and due to
> the object deconstruction above this, those fields will just be undefined
> if they weren't passed along via params.jwt or aren't a default value.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/feathersjs/feathers-authentication/pull/255/files/0edd5ccefaad58a09eed1c58d9b8f1c1f23ba6e8#r73454675>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAH3WX6lz5v2EhvQ22Kq6UcSkcWEmFB8ks5qcVBZgaJpZM4JZKVM>
> .
>